### ISSUE TITLE
Build with Xcode 15.3, run on iOS 17.4, 100% crash on launch: pb_enc_bool EXC_BAD_ACCESS (KERN_INVALID_ADDRESS)

### ISSUE THREAD

**Issue by aalenliang** [firebase/firebase-ios-sdk#12457]
### Description

Released app version 2.0 on 2/27 using Xcode 15.2, which worked smoothly. Version 2.0.1, aimed at bug fixes and built with Xcode 15.3 Release Candidate, was released on 2/28. Encountered a critical crash issue on 2/29, reported via Firebase, potentially due to the Xcode update. 

Reverting to Xcode 15.2 for version 2.0.2 resolved the issue. It suggests a possible compatibility problem with Xcode 15.3 and iOS 17.4, causing crashes at launch.

Environments:
Xcode version: Xcode 15.3 Release Candidate (15E5202a)
macOS version: macOS Sonoma 14.3.1 (23D60)
Firebase version: Firebase(10.20.0), Google-Mobile-Ads-SDK (10.14.0)

Crashlytics report:
![CleanShot 2024-03-01 at 10 30 02@2x](https://github.com/firebase/firebase-ios-sdk/assets/20056005/aaa4164d-c4a2-424a-b00d-0d31e0d0d35e)

### Reproducing the issue

1. Archive with Xcode 15.3, using Firebase 10.20.0
2. Download and open in iOS 17.4
3. Crash on launch

### Firebase SDK Version

10.20.0

### Xcode Version

Xcode 15.3 Release Candidate (15E5202a)

### Installation Method

CocoaPods

### Firebase Product(s)

Analytics, Crashlytics, Messaging, Remote Config

### Targeted Platforms

iOS

### Relevant Log Output

```shell
# Crashlytics - Stack trace
# Platform: apple
# Version: 2.0.1 (195)
# Issue: 897a934eea45e91a1448d62fff6314d1
# Session: d0ea094988b9432c9400a3bee78ebeb1_DNE_0_v2
# Date: Fri Mar 01 2024 10:05:08 GMT+0800 (China Standard Time)

Crashed: com.apple.main-thread
0  nanopb                         0x6904 pb_enc_bool + 140 (pb_encode.c:140)
1  nanopb                         0x6074 pb_encode + 544 (pb_encode.c:544)
2  FirebaseSessions               0x413c FIRSESEncodeProto + 60 (FIRSESNanoPBHelpers.m:60)
3  FirebaseSessions               0xefd4 SessionStartEvent.transportBytes() + 155 (SessionStartEvent.swift:155)
4  FirebaseSessions               0xf25c @objc SessionStartEvent.transportBytes() + 28 (<compiler-generated>:28)
5  GoogleDataTransport            0xdb34 -[GDTCOREvent setDataObject:] + 101 (GDTCOREvent.m:101)
6  GoogleDataTransport            0xd9c4 -[GDTCOREvent copy] + 69 (GDTCOREvent.m:69)
7  GoogleDataTransport            0x1aa08 -[GDTCORTransport sendEvent:onComplete:] + 89 (GDTCORTransport.m:89)
8  FirebaseSessions               0xa7bc protocol witness for GoogleDataTransportProtocol.logGDTEvent(event:completion:) in conformance GDTCORTransport + 31 (GoogleDataTransport+GoogleDataTransportProtocol.swift:31)
9  FirebaseSessions               0x5ec0 EventGDTLogger.logEvent(event:completion:) + 52 (EventGDTLogger.swift:52)
10 FirebaseSessions               0x5f44 protocol witness for EventGDTLoggerProtocol.logEvent(event:completion:) in conformance EventGDTLogger + 20 (<compiler-generated>:20)
11 FirebaseSessions               0xd14c closure #1 in SessionCoordinator.attemptLoggingSessionStart(event:callback:) + 47 (SessionCoordinator.swift:47)
12 FirebaseSessions               0xd34c closure #1 in SessionCoordinator.fillInFIID(event:callback:) + 77 (SessionCoordinator.swift:77)
13 FirebaseSessions               0xab6c partial apply for closure #1 in FIRInstallations.installationID(completion:) + 30 (Installations+InstallationsProtocol.swift:30)
14 FirebaseSessions               0xaa30 thunk for @escaping @callee_guaranteed (@guaranteed String?, @guaranteed Error?) -> () + 96 (<compiler-generated>:96)
15 FirebaseInstallations          0x49b8 __49-[FIRInstallations installationIDWithCompletion:]_block_invoke + 201 (FIRInstallations.m:201)
16 FBLPromises                    0x5130 __56-[FBLPromise chainOnQueue:chainedFulfill:chainedReject:]_block_invoke.18 + 273 (FBLPromise.m:273)
17 libdispatch.dylib              0x213c <redacted> + 32
18 libdispatch.dylib              0x3dd4 <redacted> + 20
19 libdispatch.dylib              0x125ec <redacted> + 1060
20 libdispatch.dylib              0x121b8 _dispatch_main_queue_callback_4CF + 44
21 CoreFoundation                 0x3751c <redacted> + 16
22 CoreFoundation                 0x34218 <redacted> + 1996
23 CoreFoundation                 0x33968 CFRunLoopRunSpecific + 608
24 GraphicsServices               0x34e0 GSEventRunModal + 164
25 UIKitCore                      0x22aedc <redacted> + 888
26 UIKitCore                      0x22a518 UIApplicationMain + 340
27 SwiftUI                        0x1033860 <redacted> + 600
28 SwiftUI                        0x10336a8 <redacted> + 160
29 SwiftUI                        0xc4f9fc <redacted> + 364
30 MyApp                          0x5a55c main + 4309345628 (MyAppApp.swift:4309345628)
31 ???                            0x1bc38bd84 (Missing)

Thread
0  libsystem_kernel.dylib         0xa084 __workq_kernreturn + 8
1  libsystem_pthread.dylib        0x1f6c _pthread_wqthread + 364
2  libsystem_pthread.dylib        0x1fc0 start_wqthread + 8

Thread
0  libsystem_kernel.dylib         0xa084 __workq_kernreturn + 8
1  libsystem_pthread.dylib        0x1f6c _pthread_wqthread + 364
2  libsystem_pthread.dylib        0x1fc0 start_wqthread + 8

com.google.fira.worker
0  libsystem_kernel.dylib         0x1960 _kernelrpc_mach_vm_purgable_control_trap + 8
1  libsystem_kernel.dylib         0x2f04 mach_vm_purgable_control + 40
2  libsqlite3.dylib               0x1037d0 sqlite3_sourceid + 468752
3  libsqlite3.dylib               0x7d990 sqlite3_db_release_memory + 204
4  libsqlite3.dylib               0x156dc sqlite3_finalize + 380
5  libsqlite3.dylib               0x4d5c4 sqlite3_reset + 68
6  MyApp                          0x2ab2d4 -[APMSqliteStore cleanUpStatement:forQuery:] + 4311773908
7  MyApp                          0x2aae30 -[APMSqliteStore numberForQuery:parameterValues:error:] + 4311772720
8  MyApp                          0x2a9a74 -[APMSqliteStore rowCountForTableWithName:columnName:columnValue:error:] + 4311767668
9  MyApp                          0x22dd28 -[APMDatabase upsertDataType:inTableWithName:columnName:columnValue:tableLimit:error:] + 4311260456
10 MyApp                          0x228338 -[APMDatabase updateUserAttribute:error:] + 4311237432
11 MyApp                          0x251694 -[APMMeasurement setUserPropertyInDatabaseOnWorkerQueue:] + 4311406228
12 MyApp                          0x2511bc -[APMMeasurement setUserAttributeOnWorkerQueue:] + 4311404988
13 MyApp                          0x24c500 -[APMMeasurement reportSessionStartOnWorkerQueueWithTimestamp:appInBackground:] + 4311385344
14 MyApp                          0x24c9f0 -[APMMeasurement reportFirstOpenOnWorkerQueue] + 4311386608
15 MyApp                          0x24bf28 __47-[APMMeasurement startMeasurementOnWorkerQueue]_block_invoke + 4311383848
16 MyApp                          0x24b870 -[APMMeasurement startMeasurementOnWorkerQueue] + 4311382128
17 MyApp                          0x24b13c -[APMMeasurement setEnabledOnWorkerQueue:] + 4311380284
18 MyApp                          0x24b054 __29-[APMMeasurement setEnabled:]_block_invoke + 4311380052
19 MyApp                          0x2a1fe4 __51-[APMScheduler scheduleOnWorkerQueueBlockID:block:]_block_invoke + 4311736292
20 libdispatch.dylib              0x213c <redacted> + 32
21 libdispatch.dylib              0x3dd4 <redacted> + 20
22 libdispatch.dylib              0xb400 <redacted> + 748
23 libdispatch.dylib              0xbf30 <redacted> + 380
24 libdispatch.dylib              0x16cb4 <redacted> + 288
25 libdispatch.dylib              0x16528 <redacted> + 404
26 libsystem_pthread.dylib        0x1f20 _pthread_wqthread + 288
27 libsystem_pthread.dylib        0x1fc0 start_wqthread + 8

Thread
0  libsystem_kernel.dylib         0xa084 __workq_kernreturn + 8
1  libsystem_pthread.dylib        0x1f6c _pthread_wqthread + 364
2  libsystem_pthread.dylib        0x1fc0 start_wqthread + 8

Thread
0  libsystem_kernel.dylib         0xa084 __workq_kernreturn + 8
1  libsystem_pthread.dylib        0x1f6c _pthread_wqthread + 364
2  libsystem_pthread.dylib        0x1fc0 start_wqthread + 8

com.apple.uikit.eventfetch-thread
0  libsystem_kernel.dylib         0x1af8 mach_msg2_trap + 8
1  libsystem_kernel.dylib         0x1890 mach_msg2_internal + 80
2  libsystem_kernel.dylib         0x17a8 mach_msg_overwrite + 436
3  libsystem_kernel.dylib         0x15e8 mach_msg + 24
4  CoreFoundation                 0x3601c <redacted> + 160
5  CoreFoundation                 0x33f04 <redacted> + 1208
6  CoreFoundation                 0x33968 CFRunLoopRunSpecific + 608
7  Foundation                     0x2b4a8 <redacted> + 212
8  Foundation                     0x554e8 <redacted> + 64
9  UIKitCore                      0x18dac8 <redacted> + 420
10 Foundation                     0x9ca9c <redacted> + 732
11 libsystem_pthread.dylib        0x2a90 _pthread_start + 136
12 libsystem_pthread.dylib        0x1fcc thread_start + 8

com.google.firebase.crashlytics.startup
0  libsystem_malloc.dylib         0x20e74 <redacted> + 64
1  libobjc.A.dylib                0xb738 _objc_rootAllocWithZone + 48
2  Foundation                     0x23b1c <redacted> + 64
3  Foundation                     0x23698 <redacted> + 256
4  FirebaseCrashlytics            0x13784 FIRCLSFileReadSections + 639 (FIRCLSFile.m:639)
5  FirebaseCrashlytics            0x24e60 -[FIRCLSSymbolResolver loadBinaryImagesFromFile:] + 49 (FIRCLSSymbolResolver.m:49)
6  FirebaseCrashlytics            0x1ea48 -[FIRCLSProcessReportOperation main] + 104 (FIRCLSProcessReportOperation.m:104)
7  Foundation                     0x6f594 <redacted> + 16
8  Foundation                     0x6e684 <redacted> + 648
9  FirebaseCrashlytics            0x21d18 __91-[FIRCLSReportUploader prepareAndSubmitReport:dataCollectionToken:asUrgent:withProcessing:]_block_invoke + 126 (FIRCLSReportUploader.m:126)
10 FirebaseCrashlytics            0x5210 FIRCLSApplicationActivity + 231 (FIRCLSApplication.m:231)
11 FirebaseCrashlytics            0x21b08 -[FIRCLSReportUploader prepareAndSubmitReport:dataCollectionToken:asUrgent:withProcessing:] + 163 (FIRCLSReportUploader.m:163)
12 FirebaseCrashlytics            0x10c8c __92-[FIRCLSExistingReportManager processExistingActiveReportPath:dataCollectionToken:asUrgent:]_block_invoke_2 + 244 (FIRCLSExistingReportManager.m:244)
13 Foundation                     0x437f4 <redacted> + 24
14 Foundation                     0x71d0c <redacted> + 104
15 Foundation                     0x6f594 <redacted> + 16
16 Foundation                     0x6e684 <redacted> + 648
17 Foundation                     0x7119c <redacted> + 16
18 Foundation                     0x70f08 <redacted> + 172
19 libdispatch.dylib              0x12e5c <redacted> + 148
20 libdispatch.dylib              0x3dd4 <redacted> + 20
21 libdispatch.dylib              0xb400 <redacted> + 748
22 libdispatch.dylib              0xbf30 <redacted> + 380
23 libdispatch.dylib              0x16cb4 <redacted> + 288
24 libdispatch.dylib              0x16528 <redacted> + 404
25 libsystem_pthread.dylib        0x1f20 _pthread_wqthread + 288
26 libsystem_pthread.dylib        0x1fc0 start_wqthread + 8

Thread
0  libsystem_kernel.dylib         0xa084 __workq_kernreturn + 8
1  libsystem_pthread.dylib        0x1f6c _pthread_wqthread + 364
2  libsystem_pthread.dylib        0x1fc0 start_wqthread + 8

com.google.firebase.crashlytics.MachExceptionServer
0  FirebaseCrashlytics            0x1d6b8 FIRCLSProcessRecordAllThreads + 393 (FIRCLSProcess.c:393)
1  FirebaseCrashlytics            0x1da98 FIRCLSProcessRecordAllThreads + 424 (FIRCLSProcess.c:424)
2  FirebaseCrashlytics            0x14bb8 FIRCLSHandler + 34 (FIRCLSHandler.m:34)
3  FirebaseCrashlytics            0x1743c FIRCLSMachExceptionServer + 521 (FIRCLSMachException.c:521)
4  libsystem_pthread.dylib        0x2a90 _pthread_start + 136
5  libsystem_pthread.dylib        0x1fcc thread_start + 8

Thread
0  libsystem_kernel.dylib         0xa084 __workq_kernreturn + 8
1  libsystem_pthread.dylib        0x1f6c _pthread_wqthread + 364
2  libsystem_pthread.dylib        0x1fc0 start_wqthread + 8

Thread
0  libsystem_kernel.dylib         0xa084 __workq_kernreturn + 8
1  libsystem_pthread.dylib        0x1f6c _pthread_wqthread + 364
2  libsystem_pthread.dylib        0x1fc0 start_wqthread + 8

com.apple.CFNetwork.Connection
0  libobjc.A.dylib                0x3c80 objc_msgSend + 128
1  Network                        0x298e8 <redacted> + 180
2  Network                        0x27518 <redacted> + 628
3  Network                        0x26ea0 <redacted> + 2508
4  Network                        0x26290 nw_path_create_evaluator_for_endpoint + 68
5  Network                        0x25410 <redacted> + 116
6  Network                        0x245f0 nw_association_register + 244
7  Network                        0x23414 <redacted> + 216
8  Network                        0x22960 <redacted> + 1492
9  Network                        0x89de90 <redacted> + 1084
10 Network                        0x75f8c <redacted> + 1496
11 Network                        0x4c8420 <redacted> + 11272
12 Network                        0x229a0 <redacted> + 1556
13 Network                        0x60b80 <redacted> + 1428
14 Network                        0x8d24d4 <redacted> + 6008
15 Network                        0xa1624 <redacted> + 3420
16 Network                        0x4c7304 <redacted> + 6892
17 Network                        0x229a0 <redacted> + 1556
18 Network                        0x22038 <redacted> + 960
19 Network                        0x96ed4 nw_connection_start + 496
20 CFNetwork                      0xbd254 CFURLDownloadStart + 148152
21 CFNetwork                      0xb5264 CFURLDownloadStart + 115400
22 CFNetwork                      0xb6260 CFURLDownloadStart + 119492
23 CFNetwork                      0x13a324 _CFStreamErrorFromCFError + 378972
24 CFNetwork                      0xb4d44 CFURLDownloadStart + 114088
25 CFNetwork                      0x13ad18 _CFStreamErrorFromCFError + 381520
26 CFNetwork                      0x139698 _CFStreamErrorFromCFError + 375760
27 CFNetwork                      0x139834 _CFStreamErrorFromCFError + 376172
28 libdispatch.dylib              0x213c <redacted> + 32
29 libdispatch.dylib              0x3dd4 <redacted> + 20
30 libdispatch.dylib              0xb400 <redacted> + 748
31 libdispatch.dylib              0xbf64 <redacted> + 432
32 libdispatch.dylib              0xd284 <redacted> + 1756
33 libdispatch.dylib              0x16cb4 <redacted> + 288
34 libdispatch.dylib              0x16528 <redacted> + 404
35 libsystem_pthread.dylib        0x1f20 _pthread_wqthread + 288
36 libsystem_pthread.dylib        0x1fc0 start_wqthread + 8

Thread
0  libsystem_kernel.dylib         0xa084 __workq_kernreturn + 8
1  libsystem_pthread.dylib        0x1f6c _pthread_wqthread + 364
2  libsystem_pthread.dylib        0x1fc0 start_wqthread + 8

com.apple.NSURLConnectionLoader
0  libsystem_kernel.dylib         0x1af8 mach_msg2_trap + 8
1  libsystem_kernel.dylib         0x1890 mach_msg2_internal + 80
2  libsystem_kernel.dylib         0x17a8 mach_msg_overwrite + 436
3  libsystem_kernel.dylib         0x15e8 mach_msg + 24
4  CoreFoundation                 0x3601c <redacted> + 160
5  CoreFoundation                 0x33f04 <redacted> + 1208
6  CoreFoundation                 0x33968 CFRunLoopRunSpecific + 608
7  CFNetwork                      0x25ac48 estimatedPropertyListSize + 38228
8  Foundation                     0x9ca9c <redacted> + 732
9  libsystem_pthread.dylib        0x2a90 _pthread_start + 136
10 libsystem_pthread.dylib        0x1fcc thread_start + 8
```

### If using Swift Package Manager, the project's Package.resolved

<details>
<summary>Expand <code>Package.resolved</code> snippet</summary>
<br>

```json

Replace this line with the contents of your Package.resolved.

```

</details>

### If using CocoaPods, the project's Podfile.lock

<details>
<summary>Expand <code>Podfile.lock</code> snippet</summary>
<br>

```yml

PODS:
  - Alamofire (5.5.0)
  - AppsFlyerFramework (6.5.2):
    - AppsFlyerFramework/Main (= 6.5.2)
  - AppsFlyerFramework/Main (6.5.2)
  - ClockHandRotationKit (0.0.1)
  - DefaultsKit (0.2.0)
  - DeviceKit (4.5.2)
  - EFQRCode (6.2.1):
    - swift_qrcodejs (~> 2.2.2)
  - Firebase/Analytics (10.20.0):
    - Firebase/Core
  - Firebase/Core (10.20.0):
    - Firebase/CoreOnly
    - FirebaseAnalytics (~> 10.20.0)
  - Firebase/CoreOnly (10.20.0):
    - FirebaseCore (= 10.20.0)
  - Firebase/Crashlytics (10.20.0):
    - Firebase/CoreOnly
    - FirebaseCrashlytics (~> 10.20.0)
  - Firebase/RemoteConfig (10.20.0):
    - Firebase/CoreOnly
    - FirebaseRemoteConfig (~> 10.20.0)
  - FirebaseABTesting (10.20.0):
    - FirebaseCore (~> 10.0)
  - FirebaseAnalytics (10.20.0):
    - FirebaseAnalytics/AdIdSupport (= 10.20.0)
    - FirebaseCore (~> 10.0)
    - FirebaseInstallations (~> 10.0)
    - GoogleUtilities/AppDelegateSwizzler (~> 7.11)
    - GoogleUtilities/MethodSwizzler (~> 7.11)
    - GoogleUtilities/Network (~> 7.11)
    - "GoogleUtilities/NSData+zlib (~> 7.11)"
    - nanopb (< 2.30910.0, >= 2.30908.0)
  - FirebaseAnalytics/AdIdSupport (10.20.0):
    - FirebaseCore (~> 10.0)
    - FirebaseInstallations (~> 10.0)
    - GoogleAppMeasurement (= 10.20.0)
    - GoogleUtilities/AppDelegateSwizzler (~> 7.11)
    - GoogleUtilities/MethodSwizzler (~> 7.11)
    - GoogleUtilities/Network (~> 7.11)
    - "GoogleUtilities/NSData+zlib (~> 7.11)"
    - nanopb (< 2.30910.0, >= 2.30908.0)
  - FirebaseCore (10.20.0):
    - FirebaseCoreInternal (~> 10.0)
    - GoogleUtilities/Environment (~> 7.12)
    - GoogleUtilities/Logger (~> 7.12)
  - FirebaseCoreExtension (10.20.0):
    - FirebaseCore (~> 10.0)
  - FirebaseCoreInternal (10.20.0):
    - "GoogleUtilities/NSData+zlib (~> 7.8)"
  - FirebaseCrashlytics (10.20.0):
    - FirebaseCore (~> 10.5)
    - FirebaseInstallations (~> 10.0)
    - FirebaseSessions (~> 10.5)
    - GoogleDataTransport (~> 9.2)
    - GoogleUtilities/Environment (~> 7.8)
    - nanopb (< 2.30910.0, >= 2.30908.0)
    - PromisesObjC (~> 2.1)
  - FirebaseInstallations (10.20.0):
    - FirebaseCore (~> 10.0)
    - GoogleUtilities/Environment (~> 7.8)
    - GoogleUtilities/UserDefaults (~> 7.8)
    - PromisesObjC (~> 2.1)
  - FirebaseMessaging (10.20.0):
    - FirebaseCore (~> 10.0)
    - FirebaseInstallations (~> 10.0)
    - GoogleDataTransport (~> 9.3)
    - GoogleUtilities/AppDelegateSwizzler (~> 7.8)
    - GoogleUtilities/Environment (~> 7.8)
    - GoogleUtilities/Reachability (~> 7.8)
    - GoogleUtilities/UserDefaults (~> 7.8)
    - nanopb (< 2.30910.0, >= 2.30908.0)
  - FirebaseRemoteConfig (10.20.0):
    - FirebaseABTesting (~> 10.0)
    - FirebaseCore (~> 10.0)
    - FirebaseInstallations (~> 10.0)
    - FirebaseSharedSwift (~> 10.0)
    - GoogleUtilities/Environment (~> 7.8)
    - "GoogleUtilities/NSData+zlib (~> 7.8)"
  - FirebaseSessions (10.20.0):
    - FirebaseCore (~> 10.5)
    - FirebaseCoreExtension (~> 10.0)
    - FirebaseInstallations (~> 10.0)
    - GoogleDataTransport (~> 9.2)
    - GoogleUtilities/Environment (~> 7.10)
    - nanopb (< 2.30910.0, >= 2.30908.0)
    - PromisesSwift (~> 2.1)
  - FirebaseSharedSwift (10.20.0)
  - Google-Mobile-Ads-SDK (10.14.0):
    - GoogleAppMeasurement (< 11.0, >= 7.0)
    - GoogleUserMessagingPlatform (>= 1.1)
  - GoogleAppMeasurement (10.20.0):
    - GoogleAppMeasurement/AdIdSupport (= 10.20.0)
    - GoogleUtilities/AppDelegateSwizzler (~> 7.11)
    - GoogleUtilities/MethodSwizzler (~> 7.11)
    - GoogleUtilities/Network (~> 7.11)
    - "GoogleUtilities/NSData+zlib (~> 7.11)"
    - nanopb (< 2.30910.0, >= 2.30908.0)
  - GoogleAppMeasurement/AdIdSupport (10.20.0):
    - GoogleAppMeasurement/WithoutAdIdSupport (= 10.20.0)
    - GoogleUtilities/AppDelegateSwizzler (~> 7.11)
    - GoogleUtilities/MethodSwizzler (~> 7.11)
    - GoogleUtilities/Network (~> 7.11)
    - "GoogleUtilities/NSData+zlib (~> 7.11)"
    - nanopb (< 2.30910.0, >= 2.30908.0)
  - GoogleAppMeasurement/WithoutAdIdSupport (10.20.0):
    - GoogleUtilities/AppDelegateSwizzler (~> 7.11)
    - GoogleUtilities/MethodSwizzler (~> 7.11)
    - GoogleUtilities/Network (~> 7.11)
    - "GoogleUtilities/NSData+zlib (~> 7.11)"
    - nanopb (< 2.30910.0, >= 2.30908.0)
  - GoogleDataTransport (9.3.0):
    - GoogleUtilities/Environment (~> 7.7)
    - nanopb (< 2.30910.0, >= 2.30908.0)
    - PromisesObjC (< 3.0, >= 1.2)
  - GoogleUserMessagingPlatform (2.1.0)
  - GoogleUtilities/AppDelegateSwizzler (7.12.0):
    - GoogleUtilities/Environment
    - GoogleUtilities/Logger
    - GoogleUtilities/Network
  - GoogleUtilities/Environment (7.12.0):
    - PromisesObjC (< 3.0, >= 1.2)
  - GoogleUtilities/Logger (7.12.0):
    - GoogleUtilities/Environment
  - GoogleUtilities/MethodSwizzler (7.12.0):
    - GoogleUtilities/Logger
  - GoogleUtilities/Network (7.12.0):
    - GoogleUtilities/Logger
    - "GoogleUtilities/NSData+zlib"
    - GoogleUtilities/Reachability
  - "GoogleUtilities/NSData+zlib (7.12.0)"
  - GoogleUtilities/Reachability (7.12.0):
    - GoogleUtilities/Logger
  - GoogleUtilities/UserDefaults (7.12.0):
    - GoogleUtilities/Logger
  - Hue (5.0.0)
  - Introspect (0.1.4)
  - KeychainAccess (4.2.2)
  - Kingfisher (7.6.2)
  - Localize-Swift (3.2.0):
    - Localize-Swift/LocalizeSwiftCore (= 3.2.0)
    - Localize-Swift/UIKit (= 3.2.0)
  - Localize-Swift/LocalizeSwiftCore (3.2.0)
  - Localize-Swift/UIKit (3.2.0):
    - Localize-Swift/LocalizeSwiftCore
  - MDKit (0.1.9):
    - MDKit/Core (= 0.1.9)
  - MDKit/Core (0.1.9):
    - MDKit/Foundation
  - MDKit/Foundation (0.1.9)
  - MDProducts (0.1.13)
  - nanopb (2.30909.1):
    - nanopb/decode (= 2.30909.1)
    - nanopb/encode (= 2.30909.1)
  - nanopb/decode (2.30909.1)
  - nanopb/encode (2.30909.1)
  - ObjcExceptionBridging (1.0.1):
    - ObjcExceptionBridging/ObjcExceptionBridging (= 1.0.1)
  - ObjcExceptionBridging/ObjcExceptionBridging (1.0.1)
  - PromisesObjC (2.3.1)
  - PromisesSwift (2.3.1):
    - PromisesObjC (= 2.3.1)
  - ReachabilitySwift (5.0.0)
  - SnapKit (5.6.0)
  - SwiftyJSON (5.0.1)
  - UIImageColors (2.1.0)
  - XCGLogger (7.0.1):
    - XCGLogger/Core (= 7.0.1)
  - XCGLogger/Core (7.0.1):
    - ObjcExceptionBridging

DEPENDENCIES:
  - Alamofire
  - AppsFlyerFramework
  - DefaultsKit
  - DeviceKit
  - EFQRCode
  - Firebase/Analytics
  - Firebase/Crashlytics
  - Firebase/RemoteConfig
  - FirebaseMessaging
  - Google-Mobile-Ads-SDK
  - Hue
  - Introspect
  - KeychainAccess
  - Kingfisher
  - Localize-Swift
  - MDKit
  - MDProducts
  - ReachabilitySwift
  - SnapKit
  - SwiftyJSON
  - UIImageColors
  - XCGLogger

  trunk:
    - Alamofire
    - AppsFlyerFramework
    - DefaultsKit
    - DeviceKit
    - EFQRCode
    - Firebase
    - FirebaseABTesting
    - FirebaseAnalytics
    - FirebaseCore
    - FirebaseCoreExtension
    - FirebaseCoreInternal
    - FirebaseCrashlytics
    - FirebaseInstallations
    - FirebaseMessaging
    - FirebaseRemoteConfig
    - FirebaseSessions
    - FirebaseSharedSwift
    - Google-Mobile-Ads-SDK
    - GoogleAppMeasurement
    - GoogleDataTransport
    - GoogleUserMessagingPlatform
    - GoogleUtilities
    - Hue
    - Introspect
    - KeychainAccess
    - Kingfisher
    - Localize-Swift
    - nanopb
    - ObjcExceptionBridging
    - PromisesObjC
    - PromisesSwift
    - ReachabilitySwift
    - SnapKit
    - SwiftyJSON
    - UIImageColors
    - XCGLogger

SPEC CHECKSUMS:
  Alamofire: 1c4fb5369c3fe93d2857c780d8bbe09f06f97e7c
  AppsFlyerFramework: 9f304d91cc80b6579b1206a59220383056b58b47
  DefaultsKit: 6c767941b2c3fe34c4d70e300e6c3b80c94964dd
  DeviceKit: c622fc19f795f3e0b4d75d6d11b26604338cdab3
  EFQRCode: a4d39ec3466b68dffa71de3b5caef7c9ceefdc53
  Firebase: 10c8cb12fb7ad2ae0c09ffc86cd9c1ab392a0031
  FirebaseABTesting: 1d5d49804bcfc5fa782bc2491a8c1364e2cf7241
  FirebaseAnalytics: a2731bf3670747ce8f65368b118d18aa8e368246
  FirebaseCore: 28045c1560a2600d284b9c45a904fe322dc890b6
  FirebaseCoreExtension: 0659f035b88c5a7a15a9763c48c2e6ca8c0a2977
  FirebaseCoreInternal: efeeb171ac02d623bdaefe121539939821e10811
  FirebaseCrashlytics: 81530595edb6d99f1918f723a6c33766a24a4c86
  FirebaseInstallations: 558b1da7d65afeb996fd5c814332f013234ece4e
  FirebaseMessaging: 06c414a21b122396a26847c523d5c370f8325df5
  FirebaseRemoteConfig: b873a427a48159082361343a85649eed3f5377ea
  FirebaseSessions: 2f348975f6d1c139231c180e12194161da2e0cd6
  FirebaseSharedSwift: 2fbf73618288b7a36b2014b957745dcdd781389e
  Google-Mobile-Ads-SDK: 8f67a72f88d057335b1ac3f501e2b68ad054116e
  GoogleAppMeasurement: bb3c564c3efb933136af0e94899e0a46167466a8
  GoogleDataTransport: 57c22343ab29bc686febbf7cbb13bad167c2d8fe
  GoogleUserMessagingPlatform: dce302b8f1b84d6e945812ee7a15c3f65a102cbf
  GoogleUtilities: 0759d1a57ebb953965c2dfe0ba4c82e95ccc2e34
  Hue: c129cb67be7d093a82bbbc30ce8a96757bf6f37a
  Introspect: b62c4dd2063072327c21d618ef2bedc3c87bc366
  KeychainAccess: c0c4f7f38f6fc7bbe58f5702e25f7bd2f65abf51
  Kingfisher: 6c5449c6450c5239166510ba04afe374a98afc4f
  Localize-Swift: 6f4475136bdb0d7b2882ea3d4ea919d70142b232
  nanopb: d4d75c12cd1316f4a64e3c6963f879ecd4b5e0d5
  ObjcExceptionBridging: c30e00eb3700467e695faeea30e26e18bd445001
  PromisesObjC: c50d2056b5253dadbd6c2bea79b0674bd5a52fa4
  PromisesSwift: 28dca69a9c40779916ac2d6985a0192a5cb4a265
  ReachabilitySwift: 985039c6f7b23a1da463388634119492ff86c825
  SnapKit: e01d52ebb8ddbc333eefe2132acf85c8227d9c25
  SwiftyJSON: 2f33a42c6fbc52764d96f13368585094bfd8aa5e
  UIImageColors: d2ef3b0877d203cbb06489eeb78ea8b7788caabe
  XCGLogger: 1943831ef907df55108b0b18657953f868de973b

PODFILE CHECKSUM: 2bc2ad6d066110c03e35fcad4a34d237842a641e

COCOAPODS: 1.14.3

```

</details>

**Comment by paulb777** [comment]
Sorry about the trouble.  

This is a duplicate of #11403.  The fix is planned for the 10.22.0 release targeted to go out the week of March 5th.

In the meantime, a workaround is to use Xcode 15.2 or earlier.

**Comment by ConfusedVorlon** [comment]
Not sure what's going on with releases - but XCode doesn't seem to be finding any updates past 10.18 when using SPM.

I tried setting the commit manually to (I think) 10.22.1
be49849dcba96f2b5ee550d4eceb2c0fa27dade4

But that gives me an invalid signature when I try to notarize...

**Comment by Garvnanwani** [comment]
is this issue resolved ? Still facing this

**Comment by paulb777** [comment]
This issue is resolved in 10.22.0 and 10.23.0.  If you still have a problem, please open a new issue with the template completed.
